adoc/iv-Application/SA.adoc

```diff
@@ -95,8 +107,15 @@ https://console.aws.amazon.com/cloudshell/
 
 endif::[]
 
+ifeval::[ "{cloud}" == "GCP" ]
+
+In addition, GCP provides an easy to access https://shell.cloud.google.com/[web based command line shell] where Terraform is already pre-installed.
+
+endif::[]
+
 {cloud} provides different types of storage suitable for supporting SAP workloads, so it is important to fully understand the SAP requirements for {cloud}.
 
+// @Peter and @Stephen: I would suggest omitting the following paragraph and keep the one starting by "The suggestions from ...". The second paragraph shows the same info in the first paragraph plus some more information.
```

Comment:
yes - dont know why its there twice, the first part was the initial, the second the changed one.

Reply:
@petersatsuse I commented out the first paragraph.